iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/orgalternativenames/AffMatchOrganizationAltNameFiller.java

```diff
@@ -0,0 +1,85 @@
+package eu.dnetlib.iis.wf.affmatching.orgalternativenames;
+
+import static java.util.stream.Collectors.toList;
+
+import java.io.Serializable;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+
+/**
+ * Filler of {@link AffMatchOrganization#getAlternativeNames()} list
+ * based on a dictionary of alternative organization names.
+ * 
+ * @author madryk
+ */
+public class AffMatchOrganizationAltNameFiller implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+    
+    
+    private List<List<String>> alternativeNamesDictionary = Lists.newArrayList();
+    
+
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Fills {@link AffMatchOrganization#getAlternativeNames()} list with
+     * alternative names found in dictionary.<br/>
+     * Method is looking for alternative names in dictionary
+     * based on equality of {@link AffMatchOrganization#getName()}
+     * or any of the {@link AffMatchOrganization#getAlternativeNames()} that
+     * was already present in passed organization.
+     * 
+     * Notice that method modifies {@link AffMatchOrganization#getAlternativeNames()}
+     * of the passed organization.<br/>
+     */
+    public AffMatchOrganization fillAlternativeNames(AffMatchOrganization organization) {
+        
+        List<String> alternativeNames = fetchAlternativeNames(organization.getNames());
+        
+        alternativeNames.stream().forEach(altName -> organization.addAlternativeName(altName));
+        
+        return organization;
+    }
+    
+    
+    //------------------------ PRIVATE --------------------------
+    
+    private List<String> fetchAlternativeNames(List<String> originalNames) {
+        
+        for (List<String> alternativeNames : alternativeNamesDictionary) {
+            
+            if (containsSameOrganizationName(originalNames, alternativeNames)) {
+                return filterAlreadyPresentAlternativeNames(alternativeNames, originalNames);
```

Comment:
Do we want to interrupt after finding first match? 

If we are dealing with original English name then we will stop after finding first match from the first language package, am I right?

I am wondering about covering the scenario when affiliation contains English organization name while organization name is defined in some other language. Was there any case of finding non-english name among set of organizations?